python/ray/tune/examples/pbt_transformers/pbt_transformers.py

```diff
@@ -153,31 +152,8 @@ def get_model():
     parser.add_argument(
         "--smoke-test", action="store_true", help="Finish quickly for testing"
     )
-    parser.add_argument(
```

Comment:
I would keep the --ray-address flag just in case it is used in nightly tests

Reply:
Same as above. Will get caught by the arg parser, and not used. Behavior is the same as `ray.init()`.